src/drawer/hooks/useGetProductByProvider.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { getProductByProvider } from '../apis/getProductByProvider';
+
+export const useGetProductByProvider = ({
+  providerId,
+  page = 1,
+}: {
+  providerId: string;
+  page?: number;
+}) => {
+  return useQuery({
+    queryKey: [providerId],
```

Comment:
providerId를 해당 쿼리에서만 사용할 수도 있지만, useGetProductDetail처럼 name string(?)도 추가하는 거 어떠신가용?

```suggestion
    queryKey: ['productByProvider', providerId],
```

Reply:
`useGetProductDetail` 에서 queryKey 배열에 string을 추가한 건 `productNo`가 단순 숫자 값이기 때문에
number 타입 인자를 가지는 다른 쿼리랑 key가 겹칠 수도 있을 거 같아 추가했던 건데요!!

providerId는 string + providerId를 인자로 받는 api가 이거 외엔 없어서 굳이 추가하지 않아도 될 거 같다는 생각에 이렇게 작성했었습니다~~!

한나가 추가하는 방향을 원하는 이유가 궁금햐요

Reply:
원하는 이유는 providerId를 인자로 받는 api가 혹시나 나중에 생길 수도 있지 않을까하는 염려입니당

Reply:
납득

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ